book/content/open-source-comms/comms-channels.md

```diff
@@ -0,0 +1,34 @@
+# Communication Channels
+
+The channel(s) through which you choose to communicate with your community will have a big impact on the experience and sense of belonging for your community members.
+The choice of channels will depend on the specific requirements of your project, such as keeping the community informed of any known bugs or upcoming features, assessing the current needs from the community members, inviting comments on the management plans, facilitating the overall progress of the project, as well as having somewhere to field questions and feedback.
+
+When setting up your communication channels, there are some important things to consider such as the following:
+
+* How easy is it for the community to find these channels and subscribe/unsubscribe to them?
+* How easy is it for members of the community to find and join specific conversations they are interested in?
+* Will it be a "noisy" channel (for example, lots of notifications)?
+* Can users control or filter what kind of information they receive?
+
+There are three commonly used channels: mailing lists (such as [Topicbox](https://www.topicbox.com/), or [Google groups](https://support.google.com/mail/thread/14635045?hl=en)), community forums (such as [Vanilla Forum](https://vanillaforums.com/en/software/) or [Discourse](https://www.discourse.org/)) and Internet Relay Chat (IRC) (such as [Gitter](https://gitter.im/) or [Slack](https://app.slack.com/signin)). 
+Besides these, many communities also use platforms for weekly or monthly newsletters, blogs, and social media on [Twitter](https://twitter.com/) or [Facebook](https://www.facebook.com/).
+
+In the table below, some properties of the first three communication platforms have been highlighted, which will allow Open Source project leaders to choose the most appropriate channels for their communities.
+
+## Features of different Communication Channels
+
+| Feature | Mailing Lists | Forums | Gitter |
+| :--- | :---: | :---: | :---: |
+| Media sharing | ✅ (cumbersome) | ✅ | ✅ (cumbersome) |
+| Archiveable | ✅ | ✅ | |
+| Persistant | ✅ (up to a limit) | ✅ | |
```

Comment:
I would quibble about the "up to a limit" for mailing lists. There are mailing lists that have been active for decades with public archives that are indexed by google. For example https://mail.python.org/archives/list/python-ideas@python.org/2006/12/ from December 2006. The downside of mailing lists (mostly text content only, based on fairly old tech) also means that there are 101 tools for handling the content of a list (you could probably export a list archive as a plain text file). In comparison who knows in what format the discussions on discourse are stored and which off the shelf tool will allow me to export it from discourse and import it somewhere else.